src/hypermedia-constants.js

```diff
@@ -0,0 +1,393 @@
+export const Rels = {
```

Comment:
Should we even have a constants file?  This would still allow multiple uses of the same term, and I think in theory we could only need to use these "once", in the correct entity (at least for `classes` and `actions`, maybe not `rels`)

Reply:
Probably not,  I was attempting to self encapsulate things, and copy pasting from the constants repo was the quickest way.

I think you're right that we'd only have them defined once, and use relations based between entities to sort things out